### ISSUE TITLE
mystcraft is interfering with the web display mod and is making the mod have no texture

### ISSUE THREAD

**Issue by dylankkkrose** [XCompWiz/Mystcraft-Issues#391]

**Comment by dylankkkrose** [comment]
@XCompWiz

**Comment by dylankkkrose** [comment]
@adamsalem

**Comment by dylankkkrose** [comment]
@TimoMeijer

**Comment by dylankkkrose** [comment]
@Adaptivity

**Comment by Veovis-Muaddib** [comment]
Please provide context and examples before pinging everyone you can find across Discord and Github.

Do you have screenshots or logs that show errors about textures?

Do you have a link to the version of "web display" that you're using?

Have you encountered the issue while running only Mystcraft and "web display" with no other mods?

**Comment by dylankkkrose** [comment]
> Please provide context and examples before pinging everyone you can find across Discord and Github.
> 
> Do you have screenshots or logs that show errors about textures?
> 
> Do you have a link to the version of "web display" that you're using?
> 
> Have you encountered the issue while running only Mystcraft and "web display" with no other mods?

Yeah it says exception webdisplay:screen emitting false haste true. And it says it can’t find a variant. And this only happens when I add mystcraft.
